content/MBC2.md

```diff
@@ -15,7 +15,7 @@ Same as for MBC1, but only a total of 16 ROM banks is supported.
 The MBC2 doesn't support external RAM, instead it includes 512x4 bits
 of built-in RAM (in the MBC2 chip itself). It still requires an external
 battery to save data during power-off though. As the data consists of
-4bit values, only the lower 4 bits of the "bytes" in this memory area
+4bit values, only the lower 4 bits of the bit octets in this memory area
```

Comment:
I changed this while fixing a merge conflict, awaiting some feedback on the readability

Reply:
Looks good to me.

Reply:
Well, they are still bytes, so I wonder why they aren't called that. Despite reading from `rP1` returns only data in the lower 4 bits, we still call it a byte.

Reply:
I wanted something more explicit about the 8 bit fact since we are talking about lower and upper 4 bits, but it's okay to keep bytes